docs/README.md

```diff
@@ -1,6 +1,6 @@
 # PWABuilder Suite Documentation
 
-Welcome to the new documentation for the PWABuilder tooling suite! 
+Welcome to the new documentation for the [PWABuilder](https://www.pwabuilder.com/) tooling suite! 
```

Comment:
Maybe we can remove the `new` from this now?

Reply:
```suggestion
Welcome to the documentation for the [PWABuilder](https://www.pwabuilder.com/) tooling suite! 
```